gradio/interface.py

```diff
@@ -476,7 +477,80 @@ def __repr__(self):
         return repr
 
     def get_config_file(self):
-        return utils.get_config_file(self)
+        components = []
+        layout = {"id": 0, "children": []}
+        dependencies = []
+
+        def add_component(parent, component):
+            id = len(components) + 1
+            components.append(
+                {
+                    "id": len(components) + 1,
+                    "type": component.__class__.__name__.lower(),
+                    "props": component.get_template_context(),
+                }
+            )
+            layout_context = {"id": id}
+            if isinstance(component, BlockContext):
```

Comment:
Why is there a check of `BlockContext`?

Reply:
to add the "children" field to the config. BlockContexts have children, other Blocks don't.